app/src/js/AstroVectorGrid.js

```diff
@@ -56,6 +56,13 @@ export default L.AstroVectorGrid = L.VectorGrid.Protobuf.extend({
     });
     map.on(L.Draw.Event.CREATED, this.selectFeatures, this);
 
+    // somehow "this" is lost with nested binding of event handlers
+    let vectorGrid = this;
```

Comment:
I think if you pass `this` as the last param in the event listener, `this` will remain the VectorGrid instead of becoming the map.

Reply:
I got errors when I tried to pass "this."  It wouldn't even recognize the function when I parameterized with ```e.layer.on("click", this.selectedToCSV, this)```

Reply:
Did you try passing it into the "draw:created" listener?